### DISCUSSION TITLE
Lefthook with Conventional Commits and/or Commitlint

### DISCUSSION THREAD

**Discussion by ghost** [evilmartians/lefthook#419] [Q&A]
I really like the idea of your git utility. I am working a front-end project with a separate and dedicated backend. My teammates are very new to Git and NodeJS in general. For this first project, I would like to hook up conventional commits or commitlint to assist them.  Is there a sample of this type of implementation?  I searched Google and Bing with no luck.

**Comment by mrexox** [comment]
Hey! This is a very good point. I've added a documentation about using commitlint and/or commitzen [here](https://github.com/evilmartians/lefthook/tree/master/examples/commitlint). You can use either both or just commitlint. **Notice**: if you are using commitzen, all manipulation with git must happen from CLI, because it uses TTY (which might be inconvenient when using GUI tools). But there is at least a commitlint example that should be just validating the commit message.

There was also an issue about commitlint/commitzen tools: https://github.com/evilmartians/lefthook/issues/125. I guess it might be helpful too.
